tests/integration/components/partner/client-info/component-test.js

```diff
@@ -1,26 +1,97 @@
-// import { module, test } from 'qunit';
-// import { setupRenderingTest } from 'ember-qunit';
-// import { render } from '@ember/test-helpers';
-// import { hbs } from 'ember-cli-htmlbars';
+import {
+  module,
+  test
+} from 'qunit';
+import {
+  setupRenderingTest
+} from 'ember-qunit';
+import {
+  render
+} from '@ember/test-helpers';
+import {
+  hbs
+} from 'ember-cli-htmlbars';
+import {
+  setupMirage
+} from "ember-cli-mirage/test-support";
+import {
+  setupIntl
+} from 'ember-intl/test-support';
+import dayjs from 'dayjs';
+import relativeTime from 'dayjs/plugin/relativeTime';
+import faker from 'faker';
 
-// module('Integration | Component | partner/client-info', function(hooks) {
-//   setupRenderingTest(hooks);
+module('Integration | Component | cards/client-info', function (hooks) {
+  setupRenderingTest(hooks);
+  setupMirage(hooks);
+  setupIntl(hooks);
 
-//   test('it renders', async function(assert) {
-//     // Set any properties with this.set('myProperty', 'value');
-//     // Handle any actions with this.set('myAction', function(val) { ... });
+  hooks.beforeEach(async function () {
+    await this.server.createList('organization', 2);
+    await this.owner.lookup('service:organization').load();
+    // await this.owner.lookup('service:partner').load();
+  });
 
-//     await render(hbs`<Partner::ClientInfo />`);
+  test('Client thumbnail', async function (assert) {
+    const client = this.server.create('partnerclient');
+    this.set('client', client);
+    await render(hbs `<Partner::ClientInfo @client={{this.client}}/>`);
+    assert.dom('span[data-test-thumbnail] i').hasClass('fa-users')
+  });
 
-//     assert.equal(this.element.textContent.trim(), '');
+  test("Client has valid name", async function (assert) {
+    const client = this.server.create('partnerclient');
+    this.set('client', client);
+    await render(hbs `<Partner::ClientInfo @client={{this.client}}/>`);
+    assert.dom('div[data-test-title]').hasText(this.client.name);
+  });
```

Comment:
Currently the truncate (ellipsis) is done via css so, checking exact rendered text may not be correct so, added test to verify the styles which actually applies the ellipsis. If required, a computed prop could be added to truncate the text using script based on the no.of chars.

Reply:
Current test case is good enough for truncate detection, just one suggestion, can you set a width to the card so that it won't fail in a different test runner browser width?

```
await render(hbs`<div style="width:100px"><Partner::ClientInfo @client={{this.client}}/></div>`);
```

Reply:
sure!